node/src/rpc.rs

```diff
@@ -1,3 +1,4 @@
+// Copyright 2022 Forecasting Technologies LTD.
```

Comment:
Assume we change something in this file in the next year. Do we have to put `2022-2024` in there?

Reply:
The license checker from the other PR would write "Copyright 2022, 2024 Forecasting [...]". I think I've seen people do that before.